src/overlays/actors/ovl_En_S_Goro/z_en_s_goro.c

```diff
@@ -74,85 +144,1336 @@ static DamageTable D_80BBF858 = {
+                        this->actionFlags |= EN_S_GORO_ACTIONFLAG_TIRED;
+                        return 0x66D;
+                    }
+                    if (!(this->bombbuyFlags & EN_S_GORO_BOMBBUYFLAG_TALKED_ZORA)) {
+                        this->bombbuyFlags |= EN_S_GORO_BOMBBUYFLAG_TALKED_ZORA;
+                        return 0x664;
+                    }
+                    this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                    this->actionFlags |= EN_S_GORO_ACTIONFLAG_TIRED;
+                    return 0x66C;
+
+                case PLAYER_FORM_DEKU:
+                    if ((gSaveContext.save.day == 3) && (gSaveContext.save.isNight)) {
+                        if (!(this->bombbuyFlags & EN_S_GORO_BOMBBUYFLAG_TALKED_DEKU_FINALNIGHT)) {
+                            this->bombbuyFlags |= EN_S_GORO_BOMBBUYFLAG_TALKED_DEKU_FINALNIGHT;
+                            return 0x668;
+                        }
+                        this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                        return 0x66D;
+                    }
+                    if (!(this->bombbuyFlags & EN_S_GORO_BOMBBUYFLAG_TALKED_DEKU)) {
+                        this->bombbuyFlags |= EN_S_GORO_BOMBBUYFLAG_TALKED_DEKU;
+                        return 0x664;
+                    }
+                    this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                    return 0x66C;
+
+                case PLAYER_FORM_HUMAN:
+                    if ((gSaveContext.save.day == 3) && (gSaveContext.save.isNight)) {
+                        if (!(this->bombbuyFlags & EN_S_GORO_BOMBBUYFLAG_TALKED_HUMAN_FINALNIGHT)) {
+                            this->bombbuyFlags |= EN_S_GORO_BOMBBUYFLAG_TALKED_HUMAN_FINALNIGHT;
+                            return 0x668;
+                        }
+                        this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                        return 0x66D;
+                    }
+                    if (!(this->bombbuyFlags & EN_S_GORO_BOMBBUYFLAG_TALKED_HUMAN)) {
+                        this->bombbuyFlags |= EN_S_GORO_BOMBBUYFLAG_TALKED_HUMAN;
+                        return 0x664;
+                    }
+                    this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+                    return 0x66C;
+            }
+
+        case 0x664:
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_HANDTAP;
+            return 0x665;
+        case 0x665:
+            this->actionFlags &= ~EN_S_GORO_ACTIONFLAG_HANDTAP;
+            return 0x666;
+        case 0x666:
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_LASTMESSAGE;
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_TIRED;
+            return 0x667;
+
+        case 0x668:
+            this->actionFlags |= EN_S_GORO_ACTIONFLAG_HANDTAP;
+            return 0x669;
+        case 0x669:
+            this->actionFlags &= ~EN_S_GORO_ACTIONFLAG_HANDTAP;
+            return 0x66A;
+        case 0x66A:
```

Comment:
```suggestion
            return 0x669;
            
        case 0x669:
            this->actionFlags &= ~EN_S_GORO_ACTIONFLAG_HANDTAP;
            return 0x66A;
            
        case 0x66A:
```
There's many more of this below, could you do the rest of the switch case with a space right before the next case?